frontend/app/helpers/path-helper.js

```diff
@@ -217,26 +217,6 @@ module.exports = function() {
       return PathHelper.apiWorkPackagesPath() + '/column_sums';
     },
 
```

Comment:
@ulferts Are you aware that this is still used here: https://github.com/NobodysNightmare/openproject/blob/feature/remove_apiv2_access/frontend/app/timelines/services/timeline-loader-service.js#L607-607

I could hardcode it over there. That might be okay, because nobody would think it was okay to use v2 ever again. But then this is the path helper and we might want to keep it as long as we use it.